shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -233,17 +233,19 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
             }
             result.setMovementGranularities(granularities);
         }
-        if (object.hasAction(Action.SET_SELECTION)) {
-            result.addAction(AccessibilityNodeInfo.ACTION_SET_SELECTION);
-        }
-        if (object.hasAction(Action.COPY)) {
-            result.addAction(AccessibilityNodeInfo.ACTION_COPY);
-        }
-        if (object.hasAction(Action.CUT)) {
-            result.addAction(AccessibilityNodeInfo.ACTION_CUT);
-        }
-        if (object.hasAction(Action.PASTE)) {
-            result.addAction(AccessibilityNodeInfo.ACTION_PASTE);
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN_MR2) {
```

Comment:
We should document that this limitation exists for accessibility somewhere.  For now a wiki would probably be good enough.

Reply:
Here as well, please add a comment about what the behavioral difference is, if the API version is not met.